tests/js/notes/Notes-button.test.js

```
@@ -15,11 +15,10 @@ describe('Notes class', () => {
   beforeEach(() => {
     jest.clearAllMocks()
     Story.getEditDescriptionButtonContainer.mockResolvedValue(document.createElement('div'))
-    chrome.runtime.sendMessage.mockResolvedValue({data: 'Mocked Data'})
+    chrome.runtime.sendMessage.mockResolvedValue({ data: 'Mocked Data' })
     const originalLocation = window.location
     delete window.location
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)